cmd/orchestrate/orchestrate.go

```diff
@@ -0,0 +1,314 @@
+// The orchestrate service plays the role of processing messages
+// in stand-alone operations.
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"sda-pipeline/internal/broker"
+	"sda-pipeline/internal/config"
+
+	uuid "github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
```

Comment:
would have preferred if these queues were configurable but the default was this, as this would enable for different deployment strategies

Reply:
I can see what I can do about that in a future PR.

Reply:
ok, then i can wait for another PR about it